jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -587,6 +587,13 @@ void java_bytecode_convert_methodt::convert(
   method_symbol.location=m.source_location;
   method_symbol.location.set_function(method_identifier);
 
+  code_typet::exceptionst &exceptions_list = code_type.throws_exceptions();
+  for(const auto &p : m.throws_exception_table)
+  {
+    if(!p.empty())
+      exceptions_list.push_back(irept(p));
```

Comment:
The disadvantage of `typedef` comes into play here. If they weren't used, it would've never occurred to you to convert `irept` to `irept`.

This function could be rewritten as `std::copy_if`.